index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href='https://fonts.googleapis.com/css?family=inter' rel='stylesheet'>
+    <link rel="stylesheet" href="./style.css">
+    <title>Porfolio</title>
+</head>
+<body>
+    <header>
+        <div class="logo-container">
+            <h2>My Logo</h2>
+        </div>
+        <div class="open_mobile_menu_icon">
+            <img src="Icons/Menu.svg" alt="">
+        </div>
+    </header>
+    <section class="headline">
+        <div class="welcome_text">
+            <h2>Hey There.<br> I'm Christian.<br><span>I’m a software developer</h2></span>  
+        </div>
```

Comment:
- [ ] The main title (e.g., "I'm Marianne ...") should be in a `h1` tag. If you want that google (and other search engines) to know about a specific topic, in this case, you, use an h1 tag. Check [this link](https://www.reliablesoft.net/h1-tag/) for more info.